Mathlib/NumberTheory/ClassNumber/Finite.lean

```diff
@@ -298,7 +298,7 @@ theorem exists_mk0_eq_mk0 [IsDedekindDomain S] [Algebra.IsAlgebraic R L] (I : (I
   obtain ⟨b, b_mem, b_ne_zero, b_min⟩ := exists_min abv I
   suffices Ideal.span {b} ∣ Ideal.span {algebraMap _ _ M} * I.1 by
     obtain ⟨J, hJ⟩ := this
-    refine ⟨⟨J, ?_⟩, ?_, ?_⟩
+    refine' ⟨⟨J, _⟩, _, _⟩
```

Comment:
Backwards?

Reply:
```suggestion
    refine ⟨⟨J, ?_⟩, ?_, ?_⟩

Reply:
The refine' below is *not* removable; could this be the source of the error (that *all* `refine'`s in that proof were reverted)?

Reply:
This is not a good sign. Metavariable shenanigans might be happening here

Reply:
To follow up on this: this change is now removed from the diff: the diff in that file looks entirely reasonable.

It's very possible meta-variable shenanigans happen here - but my impression is that the current change should not perturb them.